docs/user/management.asciidoc

```diff
@@ -139,6 +139,9 @@ a| <<index-patterns, Index Patterns>>
 | Copy, edit, delete, import, and export your saved objects.
 These include dashboards, visualizations, maps, index patterns, Canvas workpads, and more.
 
+| <<managing-tags, Tags>>
```

Comment:
I think this will fail until you `include::` the new `managing-tags.asciidoc` file at the bottom of this file so it gets included in the output.

This is the current error in CI:
> WARNING: invalid reference: managing-tags

Reply:
Ah, thank you, I was trying to find what I did wrong here.